Makefile

```diff
@@ -108,9 +108,11 @@ test/gc/node_modules/weak/build/Release/weakref.node: $(NODE_EXE)
 		--directory="$(shell pwd)/test/gc/node_modules/weak" \
 		--nodedir="$(shell pwd)"
 
-build-addons: $(NODE_EXE)
+rebuild-doc-addons: $(NODE_EXE)
```

Comment:
Ideally, build-addons would take dependencies on the gyp and .cc files so that it only runs when something has changed.